### PR TITLE
Attempt to fix PhoneHomeStatsTestCase.test_performance_100 being flaky.

### LINKED ISSUE #7627: PhoneHomeStatsTestCase.test_performance_100 seems flaky on SQLite
I've seen it fail quite a few times over the past couple of days in BuildKite, exclusively on SQLite tests, with:

```
Traceback (most recent call last):
  File "/src/tests/test_phone_home.py", line 51, in test_performance_100
    self.assertApproximates(stats["cpu_average"], 100, tolerance=2.5)
  File "/src/.tox/py36/lib/python3.6/site-packages/twisted/trial/_synctest.py", line 564, in assertApproximates
    raise self.failureException(msg or "%s ~== %s" % (first, second))
twisted.trial.unittest.FailTest: 103 ~== 100
```

### PR DESCRIPTION
**Pull request by clokep**
I'm hoping this will fix #7627 (and maybe #6676, but I don't have a RaspberryPi hooked up to test that).

### CODE REVIEW

**Comment by clokep** [comment]
I should note that this uses the idea given in #6676.

File: synapse/app/homeserver.py [hunk 1]
```diff
@@ -488,6 +488,29 @@ def phone_stats_home(hs, stats, stats_process=_stats_process):
     if uptime < 0:
         uptime = 0
 
+    #
+    # Performance statistics
+    #
```
Comment (clokep) [inline synapse/app/homeserver.py:493]: I feel like this should have some comment saying to keep this at the top or something...?

Comment (richvdh) [reply synapse/app/homeserver.py:493]:
```suggestion
    #
    # Performance statistics. Keep this early in the function to maintain reliability of `test_performance_100` test.
    #
```
maybe?

Comment (clokep) [reply synapse/app/homeserver.py:493]: It's better than what I came up with...

**Pull request merged**